ios/Sources/HomeFeature/HomeScreen.swift

```diff
@@ -1,16 +1,88 @@
-import SwiftUI
+import ComposableArchitecture
 import Styleguide
+import SwiftUI
 
 public struct HomeScreen: View {
-    public init() {}
+    private let store: Store<HomeState, HomeAction>
+
+    public init(store: Store<HomeState, HomeAction>) {
+        self.store = store
+    }
 
     public var body: some View {
-        Text(L10n.HomeScreen.title)
+        ZStack(alignment: .top) {
+            NavigationView {
+                ScrollView {
+                    ZStack(alignment: .top) {
+                        Color(AssetColor.primary.color)
+                            .frame(width: nil, height: 200)
+                        WithViewStore(store) { viewStore in
+                            VStack(alignment: .trailing, spacing: 0) {
+                                Text("DroidKaigi 2021 (7/31) D-7")
+                                    .foregroundColor(Color(AssetColor.Base.white.color))
+                                    .padding(.vertical, 12)
+                                    .padding(.horizontal, 8)
+                                    .background(Color(AssetColor.primaryDark.color))
+                                    .padding(.vertical)
+                                // TODO: Replace with card(large)
```

Comment:
Because https://github.com/DroidKaigi/conference-app-2021/pull/458 is merged. You could solve this TODO if you want in this PR.

Reply:
OK. I will solve this TODOs in this PR later :pray: